styles/components/BibDetails.module.scss

```diff
@@ -0,0 +1,14 @@
+@import "../utils/mixins";
+
+.bibDetails {
+  dl > dt,
+  dl > dd {
+    border: none;
```

Comment:
Can you leave the default border to see what Apoorva thinks?

Reply:
I spoke with @apoorvakhanolkar earlier about this and she requested that it be removed.

Reply:
Here are screenshots of the two different styling options. @apoorvakhanolkar can you confirm which is preferable?
<img width="1792" alt="Screen Shot 2024-01-12 at 9 36 45 AM" src="https://github.com/NYPL/research-catalog/assets/86011349/beb545e1-857d-4907-b76c-29d7c96ba874">
<img width="1792" alt="Screen Shot 2024-01-12 at 9 36 35 AM" src="https://github.com/NYPL/research-catalog/assets/86011349/6f66d82c-06d6-4848-a642-d7c271a86de4">

Reply:
I think the border looks classy 😎 . Looks much better now with the headings too, imo.

Reply:
I prefer it without the borders! Just looks a bit cleaner to me.